src/controls/qml/IntSelector.qml

```diff
@@ -32,14 +32,26 @@ Row {
     property string unitMarker: "%"
     // initial value of the control
     property int value: 0
-    // labelWidthRatio is the width of the label with respect to the total width
-    property real labelWidthRatio: 0.42857
-    // fontToHeightRatio is the size of the font relative to the height
-    property real fontToHeightRatio: 0.3
+    // left and right margin for the row content
+    property int rowMargin: Dims.w(15)
+    // size of the icon/s
+    property int iconSize: Dims.l(20)
+    // size of the label text
+    property int labelFontSize: Dims.l(6)
+
+    width: parent.width
```

Comment:
This undoes this PR: https://github.com/AsteroidOS/qml-asteroid/pull/32.

Do we want this?

Reply:
I have no opinion yet from @dodoradio but imho the change is in line with the original idea of making the addition of those elements easy and more consistent. By keeping the ability to resize the icon and label text when calling the component
The only thing it breaks is the scalability of the whole component element, which is wanted.
Since this lead to the component only looking good at Dims.h(20) row height where it was optimized for.
Now we changed that to 25 dims row height and scaling the Intselector to the new height and width including margins gave very awkward results and clipping font into icons.
The method used in that PR assumed to make the component smaller. But not larger.
Next is the ideal icon size. It is dims 20 all over the OS. Which has been proven to work out very well in visual presentation and usability. I can not see a usecase on a watch to make the component/icon smaller

So what this implementation does is enforce the 25 dims row height and give sane default values that make things look consistent when "just using" the component in default.
The rowMargin (Left/Right), iconSize and labelFontSize can still be adjusted when really needed.